### PR TITLE
Add token unique method described in the paper

### PR DESCRIPTION
**Pull request by m00nlight**
To distinguish same words in same document as different token, add the method `normalize_words` to assign a number to the same words.

### CODE REVIEW

File: ppjoin.py [hunk 1]
```diff
@@ -85,3 +88,16 @@ def prepare_strings(list_of_strings):
...
+
+def normalize_words(words):
+	"""
+	Normalize same words in document to unique words tokens as described in
+	the paper.
+	"""
+	words.sort()
+	tmp = [list(g) for k, g in groupby(words)]
+
+	wwi = map(lambda ws: [x + "@#" + str(i) for i, x in enumerate(ws)], tmp)
```
Comment (teh) [inline ppjoin.py:101]: Maybe add a comment why you're adding @# ?

**Comment by teh** [comment]
Sorry for the delay, I was away for a few days. Thanks for the fix!

File: ppjoin.py (hunk 1 above)
Comment (m00nlight) [inline ppjoin.py:101]: for example "as soon as possible" wil become "as@#0 soon@#0 as@#1 possible@#0", just for speration for the index and the origin word itself.  No special meaning, we could also remove the seperation, just add number of index :)

**Comment by teh** [comment]
Thanks!

**Pull request merged**